recipe_scrapers/hellofresh.py

```diff
@@ -72,3 +72,11 @@ def instructions(self):
                 )
             ]
         )
+
+    def image(self):
```

Comment:
Did we just forget to add this earlier? Good catch!

Reply:
oops...no, I meant to put that on my sunbasket scraper.  hope i didn't break the hellofresh.py one.